src/main/java/uk/gov/hmcts/darts/common/entity/MediaEntity.java

```diff
@@ -136,6 +136,11 @@ public class MediaEntity extends CreatedModifiedBaseEntity
     @OneToMany(mappedBy = ExternalObjectDirectoryEntity_.MEDIA)
     private List<ExternalObjectDirectoryEntity> externalObjectDirectoryEntities = new ArrayList<>();
 
+    @Column(name = "subcontent_object_id")
+    private String subcontentObjectId;
+
+    @Column(name = "subcontent_position")
+    private Integer subcontentPosition;
```

Comment:
Validation (See previous comment)?

Reply:
See earlier response to similar comment.